### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by AleksueiR** [HaNdTriX/webextension-toolbox#11]
Would it be hard or at all possible to add the support for Typescript?

**Comment by HaNdTriX** [comment]
Right now I believe yes. Since the webpack config isn't exposed to the user.
We might change this in the future.

**Comment by HaNdTriX** [comment]
We are planning a configuration option for this in the next mayor release. Please check out https://github.com/HaNdTriX/webextension-toolbox/pull/18 for the progress on that.

**Comment by HaNdTriX** [comment]
Good news! I just released the `webextension-toolbox@1.0.0-beta.1` Version.
If you want to play around with it install it via:

```bash
npm i webextension-toolbox@next
```

https://github.com/HaNdTriX/webextension-toolbox/tree/next#customizing-webpack-config

**Comment by HaNdTriX** [comment]
We just published Version 1.0.0. You can now add typescript manually:

https://github.com/HaNdTriX/webextension-toolbox#customizing-webpack-config

### This is how it could look like:

```bash
npm i ts-loader --only=dev
```

**webextension-toolbox.js**

```js
module.exports = {
  webpack: (config, { dev, vendor }) => {

    // Add typescript loader
    config.module.rules.push({
      test: /\.tsx?$/,
      loader: 'ts-loader'
    })

    return config
  }
}
```
